website/data/docs-nav-data.json

```diff
@@ -626,6 +626,10 @@
             "title": "Technical Specifications",
             "path": "connect/cluster-peering/tech-specs"
           },
+          {
+            "title": "Configuration",
+            "path": "connect/cluster-peering/configuration"
+          },
```

Comment:
@boruszak no idea if you want this visible or `"hidden": true`. Let me know.

Seems the build system requires all docs be explicitly linked in the nav, even if they're hidden.

Reply:
looks like the issue might actually be that the 1.15.x branch contains website/content/docs/connect/cluster-peering/configuration.mdx but main doesn’t